standalone-metastore/metastore-common/pom.xml

```diff
@@ -333,6 +333,12 @@
       <artifactId>slf4j-simple</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>${junit.jupiter.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Why is this kicking in here? the junit depedency in past have led to tests being silently ignored, can you check if the total number of tests that ran didn't reduce. I suspect they did

Reply:
I needed this after moving `IPStackUtilsTest` class from `common` to `metastore-common` because tests were using some methods from junit5. Reverted this change in pom.xml and adapted the tests.